src/Microsoft.Extensions.Localization/ResourceManagerStringLocalizer.cs

```diff
@@ -197,17 +204,21 @@ private IEnumerable<string> GetResourceNamesFromCultureHierarchy(CultureInfo sta
             var currentCulture = startingCulture;
             var resourceNames = new HashSet<string>();
 
+            var hasAnyCultures = false;
```

Comment:
I don't think we strictly can, as the resource names are the names of the KVP's, and if the resx file were empty but existent then we would say they were missing a manifest, which would be wrong.